src/slices/store.ts

```diff
@@ -0,0 +1,22 @@
+import { configureStore } from '@reduxjs/toolkit';
+
+import requestsReducer from 'slices/requestsSlice';
+import methodFilterReducer from 'slices/methodFiltersSlice';
+import materialsFilterReducer from 'slices/materialsFiltersSlice';
+import statusFilterReducer from 'slices/statusFiltersSlice';
+import dropdownReducer from 'slices/dropdownSlice';
+
+export const store = configureStore({
+  reducer: {
+    requests: requestsReducer,
+    methodFilters: methodFilterReducer,
+    materialFilters: materialsFilterReducer,
+    status: statusFilterReducer,
+    dropdown: dropdownReducer,
+  },
+});
+
+// Infer the `RootState` and `AppDispatch` types from the store itself
+export type RootState = ReturnType<typeof store.getState>;
```

Comment:
```suggestion
export type RootState = ReturnType<typeof store.getState>;

declare module 'react-redux'{
  interface DefaultRootState extends RootState{}
}
```
위와 같이 선언해주면 useSelector의 기본 상태타입을 설정해줘서 useSelector 사용시 콜백함수의 인자에서 `state:Rootsate` 없이 state로도 RootState를 추론한다고 하네요 ~